.github/workflows/pr.yml

```diff
@@ -15,7 +15,7 @@ jobs:
         # monorepo we want to use 10.7.0 (earliest version the Ethereum side
         # supports) or 10.13.0 (first node 10 LTS version).
         node: [10.18.0, 10.x, 11.x, 12.x, 13.x, 14.x, 15.x, 16.x]
-        os: [windows-2019, ubuntu-18.04, ubuntu-20.04, macos-11.0]
+        os: [windows-latest, ubuntu-18.04, ubuntu-20.04, macos-11.0]
```

Comment:
Why the change to `windows-latest`? Using the `latest` build makes for potentionally unreproducable CI runs, as github can change what it targets at anytime.